README.md

```diff
@@ -1,2 +1,13 @@
-# hw01
-Repository for homework 1
+## brief biography of Hyunku
+
+Hi! I am Hyunku Kwon, a _MAPSS_ student. I was born in **South Korea**. I graduated from Yonsei University, where I majored in Public Policy and Political Science, with a special focus on political liberalism and political economy of welfare state. I was an exchange student at University of Wisconsin-Madison. For the last two years, I served my military duty in South Korea. 
+
+My research interest lies in 
+1. Political Polarization
+2. Public Opinion
+3. Political Psychology
+
+Here is a image of me
+![](https://github.com/HyunkuKwon/hw01/blob/master/KakaoTalk_20171001_221540326.jpg)
+My facebook is as follows:
+[link](https://www.facebook.com/hyunku.kwon.9)
```

Comment:
I am not actually able to follow the link. However, I am not completely sure if the link as I am viewing it here is supposed to work.